pkg/github/projects/project_items_filter.go

```diff
@@ -0,0 +1,126 @@
+package projects
+
+import (
+	"strings"
+	"time"
+
+	"github.com/araddon/dateparse"
+	"github.com/grafana/github-datasource/pkg/models"
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+// filter checks if the values match the filter criteria
+func filter(fieldValue map[string]any, filters []models.Filter) bool {
+	var conj string
+	multi := len(filters) > 1
+	allMatch := false
+	if multi {
+		conj = filters[0].Conjunction
+		if conj == "and" || conj == "" {
+			allMatch = true
+			conj = "and"
+		}
+	}
+	for _, f := range filters {
+		val := fieldValue[f.Key]
+		match := match(f.Value, val, f.OP)
+		if match && !multi {
+			return true
+		}
+		if match && conj == "or" {
+			return true
```

Comment:
It looks like this is returning true when the first match is true and conj is "or". Does this test all the conditions? If so I am confused how this function works. 🙂

Reply:
if conj is "or" only 1 match has to be true.  if conj is "and" it will require all filters to match

Reply:
added some tests